js/test/babel.js

```diff
@@ -14,13 +14,5 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
-require('babel-register')({
-  ignore: function (filename) {
-    if (filename.indexOf('node_modules') !== -1) {
-      return filename.indexOf('@parity') === -1;
-    }
-
-    return false;
-  }
-});
+require('babel-register')();
```

Comment:
Explanation: All `@parity/*` export transpiled JS, so no need to babelify them again as done before.

Reply:
Good catch.